tests/php/test-tag-and-attribute-sanitizer.php

```diff
@@ -1682,6 +1682,13 @@ static function() {
 				[],
 				[ 'invalid_element' ],
 			],
+
+			'invalid_php_pi'                               => [
```

Comment:
Please also add a test for (formerly) very-common `<?xml ... ?>` processing instruction.